scribejava-apis/src/test/java/com/github/scribejava/apis/examples/SalesforceExample.java

```diff
@@ -68,7 +82,8 @@ public static void main(String... args) throws IOException, NoSuchAlgorithmExcep
         // Sample SOQL statement
         final String queryEncoded = URLEncoder.encode("Select Id, Name from Account LIMIT 10", "UTF-8");
 
-        // Building the query URI. We've parsed the instance URL from the accessToken request.
+        // Building the query URI. We've parsed the instance URL from the
+        // accessToken request.
```

Comment:
why? Line wasn't too long. It looks like the unuseful entry in the history of this class